probert/dasd.py

```diff
@@ -0,0 +1,133 @@
+# Copyright 2020 Canonical, Ltd.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import logging
+import os
+import pyudev
+import re
+import subprocess
+
+log = logging.getLogger('probert.dasd')
+
+DASD_FORMAT = r"^format\s+:.+\s+(?P<format>\w+\s\w+)$"
+DASD_BLKSIZE = r"^blocksize\s+:\shex\s\w+\s+dec\s(?P<blksize>\d+)$"
+
+
+def _search(regex, content, groupkey, flags=None):
+    if flags is None:
+        flags = re.MULTILINE
```

Comment:
No call to this passes flags, so may as well make this unconditional?

Reply:
OK